Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
+      json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
-      rexml
-      rubocop-ast (>= 1.16.0, < 2.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.20.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.16.0)
+    rubocop-ast (1.21.0)
       parser (>= 3.1.1.0)
-    rubocop-performance (1.13.3)
+    rubocop-performance (1.14.3)
       rubocop (>= 1.7.0, < 2.0)
       rubocop-ast (>= 0.4.0)
     ruby-progressbar (1.11.0)
     rubyzip (2.3.2)
-    selenium-webdriver (4.1.0)
+    selenium-webdriver (4.4.0)
       childprocess (>= 0.5, < 5.0)
       rexml (~> 3.2, >= 3.2.5)
-      rubyzip (>= 1.2.2)
-    sidekiq (6.4.1)
-      connection_pool (>= 2.2.2)
+      rubyzip (>= 1.2.2, < 3.0)
+      websocket (~> 1.0)
+    sidekiq (6.5.6)
+      connection_pool (>= 2.2.5)
       rack (~> 2.0)
-      redis (>= 4.2.0)
+      redis (>= 4.5.0, < 5)
     simple_form (5.1.0)
       actionpack (>= 5.2)
       activemodel (>= 5.2)
-    sshkit (1.21.2)
+    sshkit (1.21.3)
       net-scp (>= 1.1.2)
       net-ssh (>= 2.8.0)
     sshkit-interactive (0.3.0)
       sshkit (~> 1.12)
-    standard (1.9.1)
-      rubocop (= 1.26.1)
-      rubocop-performance (= 1.13.3)
-    stimulus-rails (1.0.4)
+    standard (1.16.1)
+      rubocop (= 1.35.1)
+      rubocop-performance (= 1.14.3)
+    stimulus-rails (1.1.0)
       railties (>= 6.0.0)
-    strscan (3.0.1)
+    strscan (3.0.4)
```

Comment:
### [strscan](https://github.com/ruby/strscan)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.0.1 → 3.0.4

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/strscan/3.0.1/3.0.4)_]





<details>
<summary>Commits</summary>

A change of **34** commits. See the full changes on [the compare page](https://github.com/ruby/strscan/compare/v3.0.1...v3.0.4).

These are the first 10 commits:
- (6f90643) [s/RubyVM::JIT/RubyVM::MJIT/g (#27)](https://github.com/ruby/strscan/commit/6f9064309d692cdfa34b76ab16b46246f0d816da)
- (4104fae) [ci: ensure using newer version than the master Ruby](https://github.com/ruby/strscan/commit/4104fae88f9c73e187e9a1daa0f0ba5d662211ed)
- (59da361) [ci: bump version with Ruby 3.0](https://github.com/ruby/strscan/commit/59da36134dd4a5da347add2bd0768ebc935900de)
- (d09198b) [Bump version](https://github.com/ruby/strscan/commit/d09198bebb23f1e036617b8f65384e1fc08834ee)
- (83ceafc) [Remove non-US-ASCII characters from C source code (#31)](https://github.com/ruby/strscan/commit/83ceafcc525d220130a832591ac9f4ca53315ca4)
- (d0fc6c0) [Fix link to CI workflows in the status badge (#34)](https://github.com/ruby/strscan/commit/d0fc6c039b44a3596a70440ca87a0991379f165a)
- (394165e) [Add Ruby 3.1 to the CI matrix (#33)](https://github.com/ruby/strscan/commit/394165e277d43cf0a85ad3f257e8c57e6bda7d71)
- (0687ebd) [Import JRuby's strscan (#25)](https://github.com/ruby/strscan/commit/0687ebd712ba8aa842a39f72785512c2d3f94fe3)
- (229b2f8) [ci: adjust style](https://github.com/ruby/strscan/commit/229b2f8c40446fce18ed8f9c0d144cb4bc8c7b35)
- (b994bd5) [ci: enable windows-latest + Ruby 3.0](https://github.com/ruby/strscan/commit/b994bd5ace1233d917b84661fff1489d315eca49)


</details>